packages/windows/Dockerfile

```diff
@@ -0,0 +1,35 @@
+
+# https://docs.microsoft.com/en-us/visualstudio/install/build-tools-container advises tht dotnet is needed for build tools
+FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2019 AS ci-windows-conan
+
+SHELL ["powershell.exe", "-ExecutionPolicy", "Bypass", "-Command"]
+
+ARG ASWF_CONAN_VERSION
+ARG ASWF_CONAN_PYTHON_VERSION
+ENV ASWF_CONAN_VERSION=${ASWF_CONAN_VERSION} \
+    ASWF_CONAN_PYTHON_VERSION=${ASWF_CONAN_PYTHON_VERSION}
+
+ENV chocolateyUseWindowsCompression=false \
+    PYTHONIOENCODING=UTF-8
+
+
+RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1')); \
+    $env:Path += '";C:\tools\python3;C:\tools\python3\Scripts"'; \
+    choco install --no-progress --yes python3 --version=${Env:ASWF_CONAN_PYTHON_VERSION} --params '"/InstallDir:C:\tools\python3"'
```

Comment:
Should the version of Python 3 be specific to the VFX Platform CY20XX, or is this just for build purposes and won't end up in the build artifacts?